### ISSUE TITLE
Parallel projection causes rendering issues with volumes when geometry is included

### ISSUE THREAD

**Issue by aclarkequad** [Kitware/vtk-js#2513]
### High-level description
When adding an actor into a renderer that also contains a volume, the rendering of the volume is correct in perspective projection, but 'breathes' when in parallel projection.

### Steps to reproduce

Change 'PARALLEL' to see the effect
```js
import '@kitware/vtk.js/Rendering/Profiles/All';

import vtkFullScreenRenderWindow from '@kitware/vtk.js/Rendering/Misc/FullScreenRenderWindow';
import vtkVolumeMapper from '@kitware/vtk.js/Rendering/Core/VolumeMapper';
import vtkVolume from '@kitware/vtk.js/Rendering/Core/Volume';
import vtkColorTransferFunction from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction';
import vtkPiecewiseFunction from '@kitware/vtk.js/Common/DataModel/PiecewiseFunction';
import vtkPlaneSource from "@kitware/vtk.js/Filters/Sources/PlaneSource";
import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
import vtkHttpDataSetReader from '@kitware/vtk.js/IO/Core/HttpDataSetReader';
import '@kitware/vtk.js/IO/Core/DataAccessHelper/HtmlDataAccessHelper';
import '@kitware/vtk.js/IO/Core/DataAccessHelper/HttpDataAccessHelper';
import '@kitware/vtk.js/IO/Core/DataAccessHelper/JSZipDataAccessHelper';

// change to enabled/disble parallel projection
const PARALLEL = false;

const fullScreenRenderWindow = vtkFullScreenRenderWindow.newInstance({
    background: [0.3, 0, 0],
  });
const renderWindow = fullScreenRenderWindow.getRenderWindow();
const renderer = fullScreenRenderWindow.getRenderer();


const actor = vtkVolume.newInstance();
const mapper = vtkVolumeMapper.newInstance();
mapper.setSampleDistance(0.7);
actor.setMapper(mapper);

const ctfun = vtkColorTransferFunction.newInstance();
ctfun.addRGBPoint(200.0, 0.4, 0.5, 0.0);
ctfun.addRGBPoint(2000.0, 1.0, 1.0, 1.0);

const ofun = vtkPiecewiseFunction.newInstance();
ofun.addPoint(200.0, 0.0);
ofun.addPoint(120.0, 0.3);
ofun.addPoint(300.0, 0.6);
actor.getProperty().setScalarOpacity(0, ofun);

actor.getProperty().setRGBTransferFunction(0, ctfun);
actor.getProperty().setScalarOpacity(0, ofun);
actor.getProperty().setScalarOpacityUnitDistance(0, 4.5);
actor.getProperty().setInterpolationTypeToLinear();
actor.getProperty().setUseGradientOpacity(0, true);
actor.getProperty().setGradientOpacityMinimumValue(0, 15);
actor.getProperty().setGradientOpacityMinimumOpacity(0, 0.0);
actor.getProperty().setGradientOpacityMaximumValue(0, 100);
actor.getProperty().setGradientOpacityMaximumOpacity(0, 1.0);
actor.getProperty().setShade(false);
actor.getProperty().setAmbient(0.2);
actor.getProperty().setDiffuse(0.7);
actor.getProperty().setSpecular(0.3);
actor.getProperty().setSpecularPower(8.0);


async function update() {
    
    fullScreenRenderWindow.resize();

    
    const plane = vtkPlaneSource.newInstance();
    const planeMapper = vtkMapper.newInstance();
    const planeActor = vtkActor.newInstance();
    
    
    planeActor.getProperty().setRepresentation(2);
    planeActor.getProperty().setColor(1, 1, 1);
    planeMapper.setInputConnection(plane.getOutputPort());
    planeActor.setMapper(planeMapper);

    const bounds = actor.getBounds();
    planeActor.setScale(bounds[1], bounds[3], bounds[5]);
    planeActor.getProperty().setLighting(false);
    planeActor.setPosition(0, 0, 40);

    renderer.updateLightsGeometryToFollowCamera();
    renderer.getActiveCamera().setParallelProjection(PARALLEL);
    
    renderer.addVolume(actor);
    renderer.addActor(planeActor);
    
    renderer.resetCamera();
    renderWindow.render();
}
    
const reader = vtkHttpDataSetReader.newInstance();
mapper.setInputConnection(reader.getOutputPort());

reader.setUrl(
    //'https://data.kitware.com/api/v1/file/58e665158d777f16d095fc2e/download', // high-res
    'https://data.kitware.com/api/v1/file/58e79a8b8d777f16d095fcd7/download',  // low-res
    { fullPath: true, compression: 'zip', loadData: true}
  ).then(() => {
    reader.loadData().then(() => {
        update();
    })
  })
```

### Detailed behavior
Two GIFs showing the behaviour, compare the ears for example over the two images:

Perspective:
![perspectiveprojection](https://user-images.githubusercontent.com/56113917/178763994-643b50bd-5646-4c16-ab32-7677ed758b55.gif)

Parallel:
![parallelprojectioon](https://user-images.githubusercontent.com/56113917/178764009-edce1a22-e80b-44c0-85ef-70aaf833397c.gif)

### Expected behavior
The volume should render the same way without the visual artifacts in both parallel and perspective projection.

### Environment
- **vtk.js**: 25.1.3
- **OS**:  Windows 10
- **Browser**: Edge 103.0.1264.37

**Comment by finetjul** [comment]
could you quickly check if this is a recent regression ? (has it always been the case ?)

**Comment by aclarkequad** [comment]
> could you quickly check if this is a recent regression ? (has it always been the case ?)

I first encountered it in version 17.15 (from memory, around that version). Updated 20.x to see if it was fixed.

**Comment by aclarkequad** [comment]
I don't have an exact version for when it broke, as I've never seen it working 'correctly'. I first saw it in version 19.1.2 which was the first time I'd put geometry into a volume with parallel projection. Sorry I don't know if it has ever worked 'correctly'.

**Comment by aclarkequad** [comment]
Fixed by commit 5caa37424d38dff67e07cefe0a0dabe8b462cff4
